src/config/options.schema.json

```diff
@@ -1144,6 +1144,31 @@
           },
           "additionalProperties": false
         },
+        "c2po": {
+          "title": "ana.c2po",
+          "type": "object",
+          "properties": {
+            "askbase": {
+              "title": "ana.c2po.askbase",
+              "description": "If true, the C-2PO Analysis uses the MayPointTo query to infer additional disequalities.",
+              "type": "boolean",
+              "default": true
+            },
+            "precise_join": {
+              "title": "ana.c2po.precise_join",
+              "description": "If true, the C-2PO Analysis uses a more precise version of the join algorithm, by using the automaton to compute the join.",
+              "type": "boolean",
+              "default": false
```

Comment:
If it's a choice between multiple algorithms, this maybe an enum option would make more sense, e.g. `join-algorithm` with two values. Then the description could mention the time-precision tradeoff or whatever.
Right now, it's like: why wouldn't I want precise?